test/models/lookup_key_test.rb

```diff
@@ -218,10 +218,10 @@ class LookupKeyTest < ActiveSupport::TestCase
         :sc_type => 'hash',
         :value => 'a:test',
       },
-      {
-        :sc_type => 'yaml',
-        :value => '{a:test}',
-      },
+      # {
+      #   :sc_type => 'yaml',
+      #   :value => '{a:test}', # TODO: update test; apparently starting Ruby 2.7.8 this won't raise an Psych::SyntaxError. Apparently YAML/parser allows that. PKM (:
```

Comment:
I think this is libyaml related: https://github.com/theforeman/foreman/pull/9900#pullrequestreview-1731939156